angular-relative-date.js

```diff
@@ -60,7 +61,7 @@
         year = day * 365;
         delta = calculateDelta(now, date);
         if (delta > day && delta < week) {
-          date = new Date(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0, 0);
+          date = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), 0, 0, 0);
```

Comment:
Maybe do the same for now here as well?  Just want to ensure that we are dealing with the same time zone.  I'll go back and look at this this weekend.  I only spent 10-15 minutes on this PR, just wanted you to get a full sense of what was involved.